esp/services/ws_workunits/ws_wudetails.cpp

```diff
@@ -327,6 +338,25 @@ void WUDetailsVisitor::buildAttribListToReturn(IConstWUPropertiesToReturn & prop
         }
     }
 
+    // Examine propertiesToReturn to see if any thing there will require a post filter
+    if (!attributesPostFilterRequired)
+    {
+        // If singleKind is specifically requested, it must be handled by post filter
```

Comment:
Slightly confusing, and worth clarifying:  It doesn't require a post filter, but the post filter code also controls whether list items are generated as single.